docs/docs/configuration-parameters.md

```diff
@@ -28,7 +28,8 @@ The configuration file should include the following fields:
   "SOARefresh": 60,
   "SOARetry":   600,
   "SOAExpire":  86400,
-  "SOAMinttl": 60
+  "SOAMinttl": 60,
+  "ContainerIPsOn": true
```

Comment:
On the other hand, the labels do not exist. But disabling it will speed up the record generation. Also false is fine for me.